modules/gdscript/gdscript_editor.cpp

```diff
@@ -2643,6 +2643,17 @@ Error GDScriptLanguage::lookup_code(const String &p_code, const String &p_symbol
 	context.base_path = p_base_path;
 	bool isfunction = false;
 
+	if (context._class && context._class->extends_class.size() > 0) {
+		bool success;
```

Comment:
Seems fine overall, just two nitpicks:
- please initalize this bool to `false`;
- please move the definition of `bool isfunction` after the code you added, since it's used in the big switch and would be better defined above it as it was.

If you can make those changes as an amend, that would be perfect. (See [PR workflow](http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html)).